hw/ip/i2c/data/i2c.hjson

```diff
@@ -102,18 +100,20 @@
           resval: "1"
         }
       ]
+      tags: [// Updated by the hw. Exclude from write-checks.
```

Comment:
@tunghoang290780 
These are the same exclusions from your PR - I just re-arranged things a little.

Reply:
@sriyerg thanks !